### PR TITLE
add display for ui tests to jobs

### LINKED ISSUE #318: Workflows: Allow UI tests

### PR DESCRIPTION
**Pull request by jperedadnr**
Fixes #318

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -33,6 +33,8 @@ jobs:
         run: |
           ver=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout -f rta)
           if [[ $ver == *"SNAPSHOT"* ]]; then
+              export DISPLAY=:90
+              Xvfb -ac :90 -screen 0 1280x1024x24 > /dev/null 2>&1 &
               mvn -B -ntp deploy -f rta
```
Comment (abhinayagarwal) [inline .github/workflows/build.yml:38]: Since we are already running tests in the previous step, there shouldn't be a need to run them again.

Comment (jperedadnr) [reply .github/workflows/build.yml]: makes sense, fixed

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -33,7 +33,7 @@ jobs:
         run: |
           ver=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout -f rta)
           if [[ $ver == *"SNAPSHOT"* ]]; then
-              mvn -B -ntp deploy -f rta
+              mvn -B -ntp -Dmaven.test.skip=true deploy -f rta
```
Comment (abhinayagarwal) [inline .github/workflows/build.yml:36]: I was checking a [previous run](https://github.com/gluonhq/rich-text-area/actions/runs/8967621767/job/24625481389) and it seems that the "Build project" phase don't run the tests. These are actually ran in "Publish Snapshots" step.

Considering this, I am of the opinion that we should remove the previous all together and make the `build.yml` similar to `release.yml` where both tests and deployment are done using the same step.

Comment (jperedadnr) [reply .github/workflows/build.yml:36]: It does run them: https://github.com/gluonhq/rich-text-area/actions/runs/8967621767/job/24625481389#step:4:34

Comment (abhinayagarwal) [reply .github/workflows/build.yml:36]: Ah, my bad. However, they [run again](https://github.com/gluonhq/rich-text-area/actions/runs/8967621767/job/24625481389#step:5:54) in the next step.

I would prefer to remove the previous step to consistency across workflows. Thoughts?

Comment (jperedadnr) [reply .github/workflows/build.yml:36]: We can't remove `Build project`, as that is used for PRs. `Publish Snapshots` only applies when committed to main.

Comment (abhinayagarwal) [reply .github/workflows/build.yml:36]: Ah, now I see why it was divided into 2 steps to begin with. Thank you for reminding me :)

**Review by abhinayagarwal — APPROVED** [review]

**Pull request merged**